cmd/fyne_demo/screens/window.go

```diff
@@ -161,6 +171,18 @@ func loadDialogGroup(win fyne.Window) *widget.Card {
 				fileSaved(writer)
 			}, win)
 		}),
+		widget.NewButton("Simple Color Picker", func() {
```

Comment:
I don't think this needs both - just add one, the type of picker feels more like a config rather than a new widget

Reply:
I've added `dialog.Refresh` and `Advanced` bool to `ColorPickerDialog` so devs can toggle the functionality, but it may still be a good idea to demo both so they can see the difference, thoughts?

Reply:
Sure that makes sense. Should advanced be off by default? I am torn :)

Reply:
I think the default should be simple, with the option of more - it feels odd to have the default be everything and the dev has to "downgrade"